governance-exercises/poll-ex3-responses.md

```diff
@@ -0,0 +1,28 @@
+## Poll Exercise 3 Responses
+
+Here are the actions I would take:
+
+1. First, I would validate the text of the standing order to ensure it aligns with Maker's principles, even if some ambiguities exist. I would also take proactive steps to resolve the ambiguity in the text of the standing order by reaching out to Emilia privately to discuss further on Friday
```

Comment:
What are your criteria here? Which principles are you judging it against?

Reply:
Worth noting it my not be possible to make changes to the standing order at that point. I suspect that they have lawyers involved in some step of the process.